apps/node_monitoring/src/monitors/delegate.rs

```diff
@@ -7,6 +7,7 @@ use std::{collections::HashMap, net::SocketAddr, time::Duration};
 
 use anyhow::Result;
 use crypto::hash::BlockHash;
+use itertools::Itertools;
```

Comment:
Why do we need it?

Reply:
We need it for joining iterator of strings.